### DISCUSSION TITLE
Do we have the separate metrics for each namespace

### DISCUSSION THREAD

**Discussion by rex0654335** [apache/kvrocks#726] [Q&A]
We are using the namespace to separate data between users and it works well.
But the metrics from the `info command stats` didn't separate by namespace as well.
As a user, we want to know the speed of commands for each namespace. 

用了namespace 来隔离业务，这个功能很不错。
通过info command stats 来看所有namespace的执行命令统计是合起来的，
有没有独立统计的指标，适用场景：区分namespace 分别执行了哪些命令，频率是多少，方便后期定位问题

**Comment by git-hulk** [comment]
@rex0654335 Not now, I think you can convert this to issue and PR is always welcome if you're willing to do this.

**Reply by liangyuanpeng** [reply]
Have created issue https://github.com/apache/incubator-kvrocks/issues/755
